src/data-types/arrays/accessing-elements.js

```diff
@@ -4,16 +4,16 @@ const cities = ['London', 'Shanghai', 'New York', 'Delhi', 'Kuala Lumpur']
 // TODO: write code to pass the tests
 
 // Set names equal to an array containing 'Bob', 'Jane', 'Joanna' in that order
-const names = null
+const names = [ 'Bob', 'Jane', 'Joanna' ]
 
 // Set fourthCity to the 4th element in the cities array
-const fourthCity = ''
+const fourthCity = 'Delhi'
```

Comment:
This isn't the solution we were looking for. Instead, it would have been good to access the elements of from the array. Such as `cities[0]` for London